src/daemon/daemon.cpp

```diff
@@ -1353,8 +1353,7 @@ try // clang-format on
         auto& vm = it->second;
         if (!mp::utils::is_running(vm->current_state()))
         {
-            return grpc::Status(grpc::StatusCode::FAILED_PRECONDITION,
-                                fmt::format("instance \"{}\" is not running", name), "");
+            return grpc::Status(grpc::StatusCode::ABORTED, fmt::format("'{}' needs starting", name));
```

Comment:
I'd prefer message said what is current state, rather than what might need to be done. I.e. "{} is not running"

Reply:
Hmm, this is printed to the console, as I see "shell failed: 'b' needs starting" before the spinner.
That's not great UI, as nothing has failed really. Can that message be suppressed?

Reply:
@gerboland,

Sure, we can suppress the message.  I think the "Starting foo" message on the console would be enough to clue the user that something is happening and tells why the shell hasn't started.

Reply:
Ok, switched back to the original message **and** the client doesn't print this out when `grpc::StatusCode::ABORTED` is returned.